content/en/docs/contributing/crds.md

```diff
@@ -20,18 +20,11 @@ This will also update the version conversion code if needed.
 
 ## Versions
 
-cert-manager currently has 4 CRD versions in use:
+cert-manager currently has a single `v1` API version.
 
-- `v1`
-- `v1beta1` (deprecated in cert-manager `v1.4.0`, removed `v1.6.0`)
-- `v1alpha3` (deprecated in cert-manager `v1.4.0`, removed `v1.6.0`)
-- `v1alpha2` (deprecated in cert-manager `v1.4.0`, removed `v1.6.0`)
+The API types are defined in [`//pkg/apis/certmanager`](https://github.com/jetstack/cert-manager/tree/master/pkg/apis/certmanager). ACME related resources are in [`//pkg/apis/acme`](https://github.com/jetstack/cert-manager/tree/master/pkg/apis/certmanager).
```

Comment:
nit (non-blocking): Could we also change the URL to the new cert-manager repo here while we're at it? Not important though, we'll need to do that site-wide at some point.